test/e2e/specs/links.test.js

```diff
@@ -16,7 +15,6 @@ import {
  *
  * @type {string}
  */
-const SELECT_WORD_MODIFIER_KEYS = process.platform === 'darwin' ? [ 'Shift', 'Alt' ] : [ 'Shift', 'Control' ];
```

Comment:
The JSDoc comment on the preceding comment existed to complement this line of code. With this line of code removed, that JSDoc should have been removed as well.

See: #21297